.ci/Jenkinsfile

```diff
@@ -240,67 +240,7 @@ pipeline {
             }
           }
       }
-    }
-    stage('extended windows') {
-      when {
-        // Always when running builds on branches/tags
-        // Enable if extended windows support related changes.
-        beforeAgent true
-        anyOf {
-          not { changeRequest() }
-          expression { return isExtendedWindowsEnabled() && env.ONLY_DOCS == "false"}
```

Comment:
In this removed code the `extended windows` stage ran on (conditions in inclusive OR):
- any non-pull request build 
- when isExtendedWindowsEnabled() is true (let's set aside `ONLY_DOCS` for a moment)

here is the code of the function
```groovy
/**
* Wrapper to know if the build should enable the windows extended support
*/
def isExtendedWindowsEnabled() {
  return env.EXT_WINDOWS_CHANGES == "true" || params.extended_windows_ci || env.GITHUB_COMMENT?.contains('extended windows') || matchesPrLabel(label: 'ci:extended-windows')
}
```

It seems we tested:
- `EXT_WINDOWS_CHANGES` env variable
- `extended_windows_ci` build parameter
- `extended windows` in the GH comment
- `ci:extended-windows` PR label

while in the new version all these are not there: I personally never used any of those but I have to wonder if we want to keep such triggers in the buildkite version...

Maybe someone more familiar with our builds can clarify that ?

Reply:
To make it triggered by a comment or a label we need to move the extended windows workflow to another pipeline. Unfortunately there's no possibility to trigger a separate step in bk.

Reply:
After a thorough discussion with @cmacknz we decided to run extended Windows tests on a PR basis. Later we can extract the extended Windows to a separate pipeline and make it triggered by a GH comment